conn.go

```diff
@@ -738,7 +738,7 @@ func IsBadConn(err error) bool {
 		}
 
 	case // cases by experience:
-		3106,  // fatal two-task communication protocol error
+        3106,  // fatal two-task communication protocol error
```

Comment:
Pkease don't introduce unnecessary whitespace changes!

Reply:
mistake, I didnt want to add this file. Fixed